src/Makefile.am

```diff
@@ -33,8 +32,15 @@ if ENABLE_BINRELOC
 geany_LDFLAGS += -Wl,-rpath='$$ORIGIN/../lib'
 endif
 
+TM_HDRS = \
+	tagmanager/tm_source_file.h \
+	tagmanager/tm_tag.h \
+	tagmanager/tm_workspace.h \
+	tagmanager/tm_parser.h
+
 geany_includedir = $(includedir)/geany
 geany_include_HEADERS = \
+	$(TM_HDRS) \
```

Comment:
Ah, thanks, I could have thought of this :-(. I've modified the commit and repushed it with this modification.